### PR TITLE
More mobile (and tablet) responsive updates

### PR DESCRIPTION
**Pull request by xtine**
* Shorten tab descriptions (to just "Read" and "Write")
* Fix display width in tablet
* Hide attachment controls in phone size (480px and below)
* Always show edit/delete comment items in mobile/tablet view
@donjo: note I changed to `display: none` and `display: inline-block` for the comment controls as that is a more preferred and robust method

eregs/notice-and-comment#145

screen shot 2016-04-25 at 3 15 23 pm
screen shot 2016-04-25 at 3 21 55 pm
screen shot 2016-04-25 at 2 25 32 pm
screen shot 2016-04-25 at 2 28 03 pm
screen shot 2016-04-25 at 3 22 39 pm

### CODE REVIEW

**Comment by cmc333333** [comment]
This looks nice, but can we avoid the `!important`s? This adds six to the file (one was already present)

**Comment by xtine** [comment]
DERP. I was confused why specificity was being weird, just realized that it's the ordering of the media queries so I'll change that up.

**Comment by xtine** [comment]
Reordered the media queries correctly and pushed over the left margin for bottom section nav on mobile.

File: regulations/static/regulations/css/less/module/comment.less [hunk 1]
```diff
@@ -684,31 +757,28 @@ a.comment-index-review {
     }
   }
 
+  .comment-controls  {
+    display: none;
+  }
```
Comment (cmc333333) [inline regulations/static/regulations/css/less/module/comment.less]: Just to verify, very narrow browsers can still add attachments, yes?

Comment (xtine) [inline regulations/static/regulations/css/less/module/comment.less]: Typical browser width doesn't go below 500 pixels, but to be safe I just put a max device width to hide the comment controls. So now desktops that scrunch to a very narrow width should be still able to see the attachment controls but tablets/phones shouldn't be able to now.

Comment (cmc333333) [inline regulations/static/regulations/css/less/module/comment.less]: I think we want all devices to be able to attach files, though?

**Comment by cmc333333** [comment]
Excellent

**Pull request merged**